src/elements/content-sidebar/ContentSidebar.scss

```diff
@@ -27,6 +32,10 @@ $sidebarWidth: $sidebarTabsWidth + $sidebarContentWidth;
             width: $sidebarWidth;
         }
     }
+
+    &.bcs-is-wider.bcs-is-open .bcs-loading {
```

Comment:
Minor, but this could follow the convention above and be combined with the existing rule via a nested modifier:

```
&.bcs-is-open {
    &.bcs-is-wider {
        width: $sidebarIncreasedWidth;
    }
}